Gemfile.lock

```diff
@@ -325,8 +325,8 @@ GEM
       concurrent-ruby (~> 1.0)
     unicode-display_width (2.5.0)
     unicode_utils (1.4.0)
-    us_core_test_kit (0.9.1)
-      inferno_core (>= 0.4.37)
+    us_core_test_kit (0.9.2)
+      inferno_core (>= 0.5.0)
```

Comment:
I find this a little surprising on a patch release but if @Jammjammjamm is good with this (I assume it was required here), no objections.

Reply:
Before releasing 0.9.0, inferno_core dependency was updated to 0.5.0 using `bundle update` command. But the `gemspec` was not updated in that release.  I think it would be good to keep `gemspec` updated also.

Reply:
Bumping it in the gemspec wasn't strictly necessary, but it was also one of the test kits that had a problem that was addressed in 0.5.0.